roles/nf-core/defaults/main.yml

```diff
@@ -55,6 +55,9 @@ pipelines:
   - name: scrnaseq
     release: 2.3.0
     container_dir: "{{ biocontainers_dirname }}"
+  - name: demultiplex
+    release: 1.4.1
+    container_dir: "{{ biocontainers_dirname }}"
```

Comment:
Have you tested a deployment with this? This kind of implies that it will pull images and deploy configs according to `tasks/pipeline.yml` and that seems not to be the way it's set up by the sequencing-report role?

Reply:
Now I did :slightly_smiling_face: Basically seqreport is setup differently because it's not an nf-core pipeline, but the service should work either way since the path to the nextflow file is defined in `main_workflow_path` in the config (which is now fully adjusted to work with the deployment from `tasks/pipeline.yml`).